chalicelib/lib/db_v2.py

```diff
@@ -202,14 +202,25 @@ def create_event_v2(events):
     """
     if isinstance(events, str):
         events = ast.literal_eval(events)
-    event = EventTable(
-        user_id=f"{events.get('user_id')}",
-        event_date=f"{events.get('event_date')}",
-        user_name=f"{events.get('user_name')}",
-        reason=f"{events.get('reason')}",
-        hours=f"{events.get('hours')}",
+
+    if isinstance(events, list):
+        response = []
+        with EventTable.batch_write() as batch:
+            for event_data in events:
+                response.append(batch.save(_generate_event(event_data)))
+        return
```

Comment:
So if a list of events, we will only get an empty response? Isn't it possible at all to get any status of the batch job?

Reply:
From what I can find in the source it's not possible to get a more detailed status. Failures will raise an exception (after a few retries) so a 200 status from this endpoint will mean success. I think the best option is to add `list` in the client and verify (or let the user do that manually).

Reply:
Ok so as long as it's a 200 it should be OK?
So the user will at least get the message about failed events. Just not which ones. I think it's good enough